flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = "A garnix module for Uptime Kuma";
```

Comment:
All descriptions in the module should stick to the conventions of the other modules. E.g. they should all end in a period (`.`).